Form-Controls/index.html

```diff
@@ -13,15 +13,64 @@
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+    <form>
+      <!-- Customer's Name -->
+      <div class="customer-name">
+        <label for="fname">First name:</label><br>
+        <input type="text" id="fname" name="fname" required pattern=".{2,}" required><br>
+        
+        <label for="lname">Last name:</label><br>
+        <input type="text" id="lname" name="lname" required pattern=".{2,}" required><br>
+      </div>
+      
+      <!-- Customer's Email -->
+      <div class="email"> 
+        <label for="customer-email">Email:</label><br>
+        <input type="email" id="customer-email" pattern="^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$"  required>
```

Comment:
Discussed in person, but email-validation regexes are a never-ending rabbit hole, and it's also common to just check for an at-symbol and a dot.

Reply:
For example: `.+@.+\..+` (I'm sure there's reasons why _that_ is also a bad implementation!)